### ISSUE TITLE
Tooltip is not visible in the fullscreen when outside: true.

### ISSUE THREAD

**Issue by karolkolodziej** [highcharts/highcharts#13769]
#### Expected behaviour
Show tooltip in the full mode the same as in the normal mode.

#### Actual behaviour
When `outside: true` tooltip is not visible in the fullscreen mode.

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/hkjtu8xn/

#### Product version
 Highcharts JS v8.1.2 (2020-06-16)

#### Affected browser(s)
All

**Comment by pawelfus** [comment]
FYI @sebastianbochan @raf18seb

**Comment by monsieur-dc** [comment]
Hello, 

This issue is still here in Highcharts v9.0.1.

Is there a workaround ?

Regards

**Comment by KacperMadej** [comment]
As a workaround, `tooltip.outside` could be toggled when fullscreen mode is opened or closed on the menu button click.
Demo: https://jsfiddle.net/BlackLabel/hkjtu8xn/1/

**Comment by kluvin** [comment]
This is a niche use-case, but setting `tooltip.outside = true` is not ideal when useHTML is enabled on any elements that may be overlapped, per [Labels and String Formatting](https://www.highcharts.com/docs/chart-concepts/labels-and-string-formatting#html-in-highcharts). Here is an example: https://jsfiddle.net/kluvin/xthgz2jk/9

**Comment by Anakii** [comment]
@kluvin is right is i use `outside:false` I can see the tooltip but when I have a custom tooltip and `useHTML: true  ` all the CSS is gone,

**Comment by lexoring** [comment]
Hi! In version Highcharts-12.1.2 the problem is still relevant, I also described the problem #22637

**Comment by lexoring** [comment]
> As a workaround, `tooltip.outside` could be toggled when fullscreen mode is opened or closed on the menu button click. Demo: https://jsfiddle.net/BlackLabel/hkjtu8xn/1/

Unfortunately this does not solve the problem in full screen.

**Comment by KacperMadej** [comment]
The tooltip needs to have `useHTML` set to `true` and be styled with CSS for it to be able to display over any HTML elements outside of the main chart SVG. https://jsfiddle.net/BlackLabel/hkjtu8xn/2/

**Comment by nishikantparmariam** [comment]
Hi, we are facing the same issue where we have set `tooltip.outside` and `tooltip.useHTML` to true to display tooltip over y-axis label:

![Image](https://github.com/user-attachments/assets/7b9885c2-fb60-480f-8f4d-9b41a6b8c848)

But tooltip isn't visible at all in full-screen mode. https://github.com/highcharts/highcharts/issues/13769#issuecomment-814182532 workaround is ok but still in full-screen mode now tooltip is behind:

![Image](https://github.com/user-attachments/assets/d86a4686-5bfa-408b-98fa-695a30ea6079)

Can you please help with a solution here. Thanks!

**Comment by TorsteinHonsi** [comment]
> IMO, this should be treated as a limitation and when entering the fullscreen the tooltip should be updated as per the workaround. What do you think?

Yes I agree. The intension of `tooltip.outside` is to make use of screen real estate outside the chart container in cases when the container is small. When in full screen, there is no more real estate outside, so `tooltip.outside` is pointless. Toggling it together with the tooltip is a good approach in my opinion.

**Comment by karolkolodziej** [comment]
@nishikantparmariam Have you tried using the full HTML tooltip as suggested in the issue you mention- [demo](https://jsfiddle.net/BlackLabel/30apdLnc/)?
Will this work in your case?

**Comment by nishikantparmariam1** [comment]
Hi @karolkolodziej - We generally discourage internal users from applying custom CSS directly. Instead, we prefer to manage such customizations via Highcharts config itself - because this data could come from a backend or a database. Relying on additional CSS stored and applied from the frontend can be challenging to maintain for various use cases we have.

Would it be possible to provide a plugin-based solution for this?

**Comment by magdalena-gut** [comment]
@nishikantparmariam1 Hi, thank you for raising this!

If you are referring to the overlapping tooltip elements, since v12.2, we’ve implemented an experimental feature that prevents this behavior. To enable it, set `Highcharts.HTMLElement.useForeignObject` to `true`.
See the demo: https://www.highcharts.com/samples/highcharts/members/renderer-usehtml-foreignobject

**Comment by nishikantparmariam1** [comment]
Hi @magdalena-gut, this is exactly what we are looking for, is it possible to provide a plugin fix to use this feature since we can't upgrade to 12.x as we recently upgraded to 11.x - and takes a lot of QA effort from our side.

Thanks!

**Comment by magdalena-gut** [comment]
@nishikantparmariam1 unfortunately, this feature involves extensive changes across several core parts of the Highcharts library, as seen in PR #22760. It touches rendering internals, layout logic, and styling layers, so it's not something we can realistically provide as a standalone plugin for older versions.

If you decide to upgrade, our Support team will be happy to help you through the process!

**Comment by TorsteinHonsi** [comment]
@nishikantparmariam1 I'm attempting a backport plugin to v11. Here's what I've got, please let me know if this covers your case: https://jsfiddle.net/highcharts/apr91kcj/ . 

~~It's a work in progress - the tooltip is currently missing. To be continued...~~
